README.md

```diff
@@ -1,3 +1,5 @@
 # [Personal Website](https://jatin2003.github.io)
 
-## Thanks to [Brittany Chiang](https://github.com/bchiang7/bchiang7.github.io)
```

Comment:
It seems like nothing has been changed & I'm not working on it anymore (like adding blog page) as I got my site up & running 😁